source/_integrations/lg_thinq.markdown

```diff
@@ -26,12 +26,12 @@ ha_codeowners:
 
 The **LG ThinQ** integration allows you to connect LG ThinQ devices to Home Assistant. The features of this integration include:
 
-- Control LG appliances as Home Assistant entities through the [LG ThinQ Connect API](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/overview/).
+- Control LG appliances as Home Assistant entities through the [LG ThinQ Connect API](https://smartsolution.developer.lge.com/en/apiManage/thinq_connect).
```

Comment:
So, we need to get rid of those links IMHO.

Home Assistant is meant to be used by users, not developers. Yet, every single link on this page links to developer documentation.

Can we fix that?

../Frenck

Reply:
@frenck 
I removed the developer site's links.